app/src/main/java/com/example/makeiteven2/extras/OnTouchAnimation.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.makeiteven2.extras
+
+import android.content.Context
+import android.view.MotionEvent
+import android.view.View
+import android.view.animation.AnimationUtils
+import com.example.makeiteven2.R
+
+class OnTouchAnimation (private var context: Context) {
+
+    private val btPressAnimation = AnimationUtils.loadAnimation(context, R.anim.btn_pressed)
+    private val btnReleaseAnimation = AnimationUtils.loadAnimation(context, R.anim.btn_realeas)
+
+    val btnTouchAnimation = View.OnTouchListener { v, event ->
+        if (event.action == MotionEvent.ACTION_DOWN) {
+            v.startAnimation(btPressAnimation)
+            btPressAnimation.fillAfter = true
+        }
+        if (event.action == MotionEvent.ACTION_UP) {
+            v.startAnimation(btnReleaseAnimation)
+        }
+        false
```

Comment:
?
false   ?

Reply:
I don't remember why but you need it for the animation to work so it's ok